File: Zadania/Część pierwsza/1.9.cpp
#include <iostream>
#include <conio.h>
#include <locale.h>

using namespace std;

int main()
{
setlocale(LC_CTYPE,"Polish");
char b=97;
int boki=3;
float bok;
float pole=1.0;
for(int i=0;i<boki;i++)
{
cout<<"prosze podac boku " << b <<":";
cin >> bok;
if (bok<0 ){i--;b--;cout<<"podana wartosc jest niepoprawna"<<endl;}
else{pole=pole*bok;}
b++;
}
cout <<"pole prostokat to: "<<pole<<endl;
cin.get();
return 0;
}

File: Zadania/Część pierwsza/alternatywne rozwiązanie/python/1.9.py
import msvcrt as m 

bok=1.0
pole=1.0
i=0
while i < 3:
    try:
        bok = float(input("Prosze podac bok: "))
    except ValueError:
        print("podana wartosc jest niepoprawna\n")
        continue
    if bok < 0:
        
        print("podana wartosc jest niepoprawna\n")
    else:
        pole=pole*bok
        i += 1
print("pole prostokat to:",pole)
m.getch() 